arkouda/client.py

```diff
@@ -356,8 +359,14 @@ def _send_string_message(
     ValueError
         Raised if the return message is malformed JSON or is missing 1..n
         expected fields
+
+    Notes
+    -----
+    - Size is not yet utilized. It is being provided not in preparation for further development.
```

Comment:
typo here "It is being provided not in preparation for further development."

Reply:
Updated

Reply:
cool